Scripts~/windows/.import-lightweight-sdk.ps1

```diff
@@ -0,0 +1,48 @@
+param(
+	[Parameter(Mandatory)]
+	[ValidatePattern("\d+\.\d+\.\d+")]
+	[string] $LightWeightUnrealServerSdkVersion
+)
+
+$ROOT_DIR=Resolve-Path "$PSScriptRoot\..\.."
+$SOURCE_DIR="$ROOT_DIR\GameLiftPlugin\Source"
+$SERVER_SDK_DIR="GameLift-Cpp-ServerSDK-UnrealPlugin-$LightWeightUnrealServerSdkVersion";
+$SERVER_SDK_FILENAME="$SERVER_SDK_DIR.zip";
+$DESTINATION_PATH="$SOURCE_DIR\GameliftServer"
```

Comment:
This is still `GameliftServer`, there may be other places were that's used vs `GameLiftServer`